core/src/main/java/com/automation/remarks/video/recorder/ffmpeg/LinuxFFmpegRecorder.java

```diff
@@ -6,7 +6,7 @@
 public class LinuxFFmpegRecorder extends FFMpegRecorder {
     @Override
     public void start() {
-        String display = ":0.0";
+        String display = System.getProperty("recorder.display", ":0.0");
```

Comment:
I can't argue with that :)
README need to be updated as well. Maybe it better to close this pull request and open `enhancement` request?